src/library/scala/collection/immutable/TreeSeqMap.scala

```diff
@@ -346,7 +345,7 @@ object TreeSeqMap extends MapFactory[TreeSeqMap] {
   }
 
   private type Mapping[K, +V] = Map[K, (Int, V)]
-  private val Mapping = Map
+  //private val Mapping = Map
```

Comment:
Why comment?

Reply:
In my next commit of more stuff, I make it at unused.